docs/content/introduction/productvisie.md

```diff
@@ -66,38 +95,61 @@ Bij de start van dit traject hanteren we de volgende uitgangspunten:
+Naast de (nieuw te ontwikkelen) OpenAPI 3 specificatie wordt een [referentie implementatie](https://nl.wikipedia.org/wiki/Referentie-implementatie) gemaakt. De referentie implementatie heeft de volgende karakteristieken:
 
 * Wordt gelijktijdig ontwikkeld met de specificatie en (geautomatiseerde) test suite;
+* Maakt het mogelijk scenariotesten uit te voeren, wat vraagt om persistentie van testdata;
 * Bewijst dat de specificatie geïmplementeerd kan worden;
 * Zorgt er voor dat leveranciers hun eigen implementatie kunnen testen;
 * Is de defacto standaard voor andere implementaties;
 * Geeft duidelijkheid over de intentie van de specificatie waar deze ruimte biedt voor interpretaties.
 
 
+## Transitie
+
+Een BIG BANG overgang is onmogelijk. Er zal een geleidelijke transitie moeten plaatsvinden waarbij het nieuwe naast het oude bestaat. De te ontwikkelen ZDS 2.0 (en verder) is de toekomst, rekening moet worden gehouden met de huidige werkelijkheid die daarnaast moet kunnen bestaan. Idealiter in de zelfde achterliggende bronnen die op meerdere manieren worden ontsloten.
+
+In het ZDS Scrumteam ligt de focus op het ontwikkelen van de nieuwe wereld. De rol van architecten in het team bestaat o.a. uit het in de gaten houden of een transitie mogelijk blijft.
+
+
+## Centraal aanbieden      
+
+Het principe "raadplegen bij de bron" veronderstelt waar mogelijk een enkele bron die wordt bijgehouden door de verantwoordelijke en welke kan worden geraadpleegd door afnemers.
+
+Voorzien wordt dat centraal aanbieden van componenten naast decentraal gebruik van grote waarde kan zijn. Om daar te komen is het volgordelijk nodig eerst de API specificatie uit te werken.
+
+Een centrale *Zaakregistratiecomponent* (ZRC) en *Documentregistratiecomponent* (DRC) is in veel ketensamenwerkingen wellicht een welkome oplossing als alternatief van de hudige praktijk van ketenautomatisering waarbij dossiers steeds worden gekopieerd naar een volgende silo. In de praktijk betekent dit dan dat organisaties afhankelijk van het proces en de ketenpartners het proces koppelen aan een andere ZRC en DRC.
+- Binnen het DSO (zie: [Gerelateerde trajecten](#gerelateerde_trajecten)) is hier mogelijk op korte termijn behoefte aan.
+- Ook binnen het Sociaal Domein heeft deze constructie potentie.
+
+Een centrale *Zaaktypecatalogus* (ZTC) kan dienen als repository van content die voor veel gemeenten gelijk zal blijken. Voor veel zaken zal het niet nodig blijken af te wijken van de referentie. Ook hier geldt dat organisaties per proces zouden kunnen kiezen welke authoratieve bron voor Zaaktypen wordt geraadpleegd - bijvoorbeeld voor interne processen een intern ZTC en voor ketenprocessen een centrale ZTC.
+
+Wanneer componenten centraal worden aangeboden is een *beheerorganisatie* noodzakelijk.
+
```

Comment:
De gedachte van het centraal aanbieden van een _bron_  is charmant. Het centraal aanbieden van diverse _registratiecomponenten_ wat minder. Is een 'Github achtig model' denkbaar (een lokale afslag van de centrale waarheid)?

Reply:
Interessante geachte, zaaktypen zouden best in git kunnen leven, met mogelijkheid om te forken, clonen, mergen... :)